packages/slider/src/Slider.ts

```diff
@@ -38,7 +38,7 @@ import { SliderHandle } from './SliderHandle.js';
 import { streamingListener } from '@spectrum-web-components/base/src/streaming-listener.js';
 import type { NumberFormatter } from '@internationalized/number';
 
-export const variants = ['filled', 'ramp', 'range', 'tick'];
+export const variants = ['filled', 'offset', 'ramp', 'range', 'tick'];
```

Comment:
Does this require a new variant, or can it be achieved via the additions of a `fill-start` attribute/property that would also allow for the offset to start at somewhere other than the middle value of the range?